### ISSUE TITLE
Incorrect web3 typings for `bytes[]`

### ISSUE THREAD

**Issue by r0b1n** [joshstevens19/ethereum-abi-types-generator#61]
When generating web3 typings for `bytes[]` and alike, the resulting typings are incorrect. 
While `bytes` is correctly represented as `string | numbers[]` (string or array of numbers), `bytes[]` is represented as follows:

Actual: `string | number[][]` - this means `string` OR two dimensional array of numbers.
Expected: `(string | number[])[]` - array of (string or array of numbers)

This behaviour is actually tested [here](https://github.com/joshstevens19/ethereum-abi-types-generator/blob/a1625ea68e32e2e33998cccc9c52a37970997361/abi-types-generator/src/converters/typescript/common/helpers.spec.ts#L551-L558) :

```typescript
expect(
    TypeScriptHelpers.getSolidityInputTsTypeByTypeName(
        'bytes[][][]',
        Provider.web3
    )
).toEqual('string | number[][][][]');
```

I believe it should result in `(string | number[])[][][]` not `string | number[][][][]`.

**Comment by joshstevens19** [comment]
web3 is not very well supported here.. PR welcome 👍

**Comment by joshstevens19** [comment]
yes you are right web3 is not as well submitted as ethos will fix this though thanks for raising
